charts/incubator/plausible/values.yaml

```diff
@@ -42,6 +42,23 @@ envFrom:
   - secretRef:
       name: '{{ include "tc.common.names.fullname" . }}-env-secret'
 
+initContainers:
+  plausible-init:
+    image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+    envFrom:
+      - configMapRef:
+          name: '{{ include "tc.common.names.fullname" . }}-env-config'
+      - secretRef:
+          name: '{{ include "tc.common.names.fullname" . }}-env-secret'
+    command: [sh]
+    args:
+      - -c
+      - >- 
+        sleep 10 &&
+        /entrypoint.sh db createdb &&
+        /entrypoint.sh db migrate &&
+        /entrypoint.sh db init-admin
```

Comment:
Does all this need to run on every start?

I guess migrate yea.

But the other 2 might be a better idea to move them to `installContainers` ?

Reply:
Oh that's a cool feature. Didn't know about that. Are init containers guaranteed to run after install containers? Or may they run in parallel?

Reply:
https://github.com/truecharts/library-charts/blob/4c2c7ddc0d87c82a55385342832aa0c3710ec073/charts/common/templates/lib/controller/_pod.tpl#L58-L90

Install -> Upgrade -> init